modules/navier_stokes/src/linearfvkernels/LinearFVEnergyAdvection.C

```diff
@@ -18,11 +18,10 @@ InputParameters
 LinearFVEnergyAdvection::validParams()
 {
   InputParameters params = LinearFVFluxKernel::validParams();
-  params.addClassDescription(
-      "Represents the matrix and right hand side contributions of an "
-      "advection term for the energy e.g. h=int(cp dT). A user may still "
-      "override what quantity is advected, but the default is specific enthalpy.");
-  params.addParam<Real>("cp", 1, "blabla");
+  params.addClassDescription("Represents the matrix and right hand side contributions of an "
+                             "advection term for the energy e.g. h=int(cp dT). A user may still "
+                             "override what quantity is advected, but the default is temperature.");
+  params.addParam<Real>("cp", 1, "Specific heat value");
```

Comment:
needs to be a functor

Reply:
It can be a functor but then I'd like you to add a `isConstant()` check for now to make sure the user doesn't add something we don't support in this implementation

Reply:
I have added a isConstant() check to the code in the new commit